dvc/repo/__init__.py

```diff
@@ -399,6 +399,9 @@ def filter_dirs(dname, root=root):
     def active_stages(self, from_directory=None):
         import networkx as nx
 
+        if from_directory:
+            assert os.path.isdir(from_directory)
```

Comment:
Let's raise a more meaningful error. This might happen quite easily, so assert is not enough.